<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingForm</class>
 <widget class="QWidget" name="settingForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/img/img/setting.png</normaloff>:/img/img/setting.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="TabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="GeneralTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QLabel" name="LanguageLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Language:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="LanguageComboBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>English</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>简体中文</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>繁體中文</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="ThemeLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Theme:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="ThemeComboBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Light</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dark</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="FontLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Font:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="FontPushButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Font</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="AboutTab">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <widget class="QLabel" name="IconLabel">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>5</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources/resources.qrc">:/icon/icon/main.ico</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="IssueLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>500</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Issue:&amp;ensp;&lt;a href=&quot;https://github.com/nixgnauhcuy/QtComMate/issues&quot;&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;https://github.com/nixgnauhcuy/QtComMate/issues&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="LatestVersionLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>500</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Latest Version:&amp;ensp;&lt;a href=&quot;https://github.com/nixgnauhcuy/QtComMate/releases&quot;&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;https://github.com/nixgnauhcuy/QtComMate/releases&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="LicenseLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>500</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;License:&amp;ensp;&lt;a href=&quot;https://github.com/nixgnauhcuy/QtComMate/blob/main/LICENSE&quot;&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;https://github.com/nixgnauhcuy/QtComMate/blob/main/LICENSE&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="VersionLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>500</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Version: 1.1.0</string>
       </property>
      </widget>
      <widget class="QLabel" name="SubTitleLabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>500</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Simple Serial Port Tool Based on pyqt6</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
